src/tito/tagger/main.py

```diff
@@ -243,10 +239,21 @@ def _make_changelog(self):
 
                 write(fd, header)
 
-                for cmd_out in output.split("\n"):
-                    write(fd, "- ")
-                    write(fd, "\n  ".join(textwrap.wrap(cmd_out, 77)))
+                # don't die if this is a new package with no history
+                if self._changelog_msg is not None:
+                    write(fd, self._changelog_msg)
```

Comment:
Is the idea here that this is just a single line? If so, we should probably write it in with a leading "- ", it might be nicer for the user not to have to format it themselves.

If you're aiming for multi-line formatted changelogs, we should probably be looking at --changelog-file instead.